react-validated-input-component/my-app/src/ValidatedInput.js

```diff
@@ -0,0 +1,49 @@
+import './ValidatedInput.css';
+import { useState } from "react";
+
+export default function ValidatedInput () {
+  const [password, setPassword] = useState('');
+  const numReg = /[0-9]+/;
+  const upReg = /[A-Z]+/;
+  const charReg = /[!@#$%^&*()]+/;
+  const passwordLength =
```

Comment:
`passwordLength` is a strange name for this variable. Maybe it's leftover from your first implementation. Be sure to run through your code and clean it up before submitting a PR.